# глубина, [м]
float32 depth